craco.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require("path");
+
+module.exports = {
+  reactScriptsVersion: "react-scripts",
+  babel: {
+    plugins: [
+      // https://github.com/lodash/babel-plugin-lodash
+      ["lodash"],
+    ],
+  },
+  webpack: {
+    alias: {
+      // De-duplicate lodash in our bundle by ensuring all versions resolve to the same dependency
+      lodash: path.resolve(__dirname, "node_modules/lodash"),
+      "lodash-es": path.resolve(__dirname, "node_modules/lodash"),
```

Comment:
Any significant difference if the same dependency resolves to the ES version of Lodash instead of the CJS?
I'm guessing the biggest reason for the original is 3rd party libraries depending on it.

Reply:
The lodash babel plugin is explicitly written to work on `lodash`, not `lodash-es`. And `lodash-es` doesn't contain `lodash/fp` which is needed for some things.

The end result is that the lodash babel plugin cuts everything down to only including what is actually needed/used, so it's like using `lodash-es` but better.